### PR TITLE
[SMALLFIX] Removed redundant initializer in FileSystemUtils

### PR DESCRIPTION
**Pull request by HorizonNet**
Removed a redundant initializer in the *FileSystemUtils* class.

### CODE REVIEW

File: core/client/src/main/java/alluxio/client/file/FileSystemUtils.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ public static boolean waitCompleted(final FileSystem fs, final AlluxioURI uri,
         ClientContext.getConf().getLong(Constants.USER_FILE_WAITCOMPLETED_POLL_MS);
     boolean completed = false;
     long timeleft = deadline - System.currentTimeMillis();
-    long toSleep = 0;
+    long toSleep;
```
Comment (aaudiber) [inline core/client/src/main/java/alluxio/client/file/FileSystemUtils.java]: we should move this as close to its usage as possible, so just after `} else if (!completed) {`

**Comment by aaudiber** [comment]
Thanks for the improvement! I left one comment

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comment. I updated the PR accordingly.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**